src/chocolatey/infrastructure.app/services/NugetService.cs

```diff
@@ -1082,8 +1082,8 @@ public void uninstall_noop(ChocolateyConfiguration config, Action<PackageResult>
                 }
                 else
                 {
-                    IPackage installedPackage = packageManager.LocalRepository.FindPackage(packageName);
-                    if (installedPackage != null) installedPackageVersions.Add(installedPackage);
+                    SemanticVersion semanticVersion = new SemanticVersion(config.Version);

```

Comment:
We could add a try/catch here. But if it throws an error here, there will also be an error thrown a bit more up in the method at line 962: 
`SemanticVersion version = config.Version != null ? new SemanticVersion(config.Version) : null;`

I quickly checked other uses of SemanticVersion in the project, and currently none of them do a try/catch. So, at least it is consistent. :-)